### ISSUE TITLE
Can't run the project locally - likely a CommonJS typescript transpiling issue. 

### ISSUE THREAD

**Issue by ajax88** [raimohanska/ourboard#256]
## Running `yarn --cwd backend dev` results in server error

Error: 
```
Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/alexjackson/playground/scrapier/ourboard/node_modules/y-protocols/awareness.js from /Users/alexjackson/playground/scrapier/ourboard/backend/dist/backend/src/y-websocket-server/WSSharedDoc.js not supported.
Instead change the require of awareness.js in /Users/alexjackson/playground/scrapier/ourboard/backend/dist/backend/src/y-websocket-server/WSSharedDoc.js to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous> (/Users/alexjackson/playground/scrapier/ourboard/backend/dist/backend/src/y-websocket-server/WSSharedDoc.js:28:40)
    at Object.<anonymous> (/Users/alexjackson/playground/scrapier/ourboard/backend/dist/backend/src/y-websocket-server/Docs.js:4:23)
    at Object.<anonymous> (/Users/alexjackson/playground/scrapier/ourboard/backend/dist/backend/src/y-websocket-server/YWebSocketServer.js:30:16)
    at Object.<anonymous> (/Users/alexjackson/playground/scrapier/ourboard/backend/dist/backend/src/board-yjs-server.js:36:44)
    at Object.<anonymous> (/Users/alexjackson/playground/scrapier/ourboard/backend/dist/backend/src/websocket-sessions.js:7:28)
    at Object.<anonymous> (/Users/alexjackson/playground/scrapier/ourboard/backend/dist/backend/src/board-state.js:33:30)
    at Object.<anonymous> (/Users/alexjackson/playground/scrapier/ourboard/backend/dist/backend/src/server.js:32:23) {
  code: 'ERR_REQUIRE_ESM'
}
```

I'm pretty sure this is related to the most recent [changes here].(https://github.com/raimohanska/ourboard/commit/0e8c3b5dbf819a553b19efac15c99902d2f43542)

My knowledge of CommonJS vs ESM and such is pretty limited, but from what I understand I think all that needs to change is removing the `.js`. 

```
import * as awarenessProtocol from "y-protocols/awareness.js"
import * as syncProtocol from "y-protocols/sync.js"
```

to 

```
import * as awarenessProtocol from "y-protocols/awareness"
import * as syncProtocol from "y-protocols/sync"
```

That seemed to fix it for me. So I'll just move on but wanted to put this up for vis.

**Comment by raimohanska** [comment]
Oh, actually it's failing on my machine as well. Pushed a quick fix in f9981ae020d51c650a3c8b00a13ab91349fe607c. Hope this helps.

**Comment by ajax88** [comment]
> Oh, actually it's failing on my machine as well. Pushed a quick fix in [f9981ae](https://github.com/raimohanska/ourboard/commit/f9981ae020d51c650a3c8b00a13ab91349fe607c). Hope this helps.

Yup that's perfect, will close this out. Thanks!
